images/installer/nested-libvirt/README.md

```diff
@@ -0,0 +1,41 @@
+# Nested libvirt on GCE
+
+This image enables launching a libvirt cluster nested in a GCE instance, for libvirt CI testing.
+
+This image contains [nss_wrapper](https://cwrap.org/nss_wrapper.html) to execute ssh commands as
```

Comment:
nit: backticks for `nss_wrapper`?

    [`nss_wrapper`](https://cwrap.org/nss_wrapper.html)

Reply:
done